benchmark/suite/macro/Makefile

```diff
@@ -9,15 +9,19 @@ TIMING_SAMPLES ?= 10
 TEST_DIR=test
 BENCHMARK_DIR ?=fresh
 
-.PHONY: external internal all clean test-clean outputs
+.PHONY: external fine all clean test-clean outputs
 
-all: external internal
+all: external fine coarse
```

Comment:
You might consider building coarse first. Like external it will probably be faster.